### PR TITLE
branch setup

### PR DESCRIPTION
**Pull request by codingwithsurya**
# Branch Setup

**What user problem are we solving?**
Running multiple git commands through a single shell script 

**What solution does this PR provide?**
single shell script to setup branch that executes multiple git commands. The git commands that this shell script runs is as follows:

```
git stash
git checkout nextjs
git pull origin nextjs
git checkout -b branch-name

```
**Testing Methodology**
How did you test your changes and verify that existing 
functionality is not broken

 dummy branch testing

**Any other considerations**

In order to run shell script:

`chmod +x create-branch.sh`

`./create-branch.sh branch-name` where `branch-name` is the desired name of the branch you want to create.

### CODE REVIEW

File: create-branch.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+branch_name=$1
+
+# checks if branch name is provided
+if [ -z "$branch_name" ]; then
```
Comment (karkir0003) [inline create-branch.sh:6]: @yeaitsurya, can you add some description in this PR to describe how to run the shell script

**Review by karkir0003 — COMMENTED** [review]
please address my minor comment and then i'll approve

**Comment by codingwithsurya** [comment]
> please address my minor comment and then i'll approve

just fixed!

**Review by karkir0003 — APPROVED** [review]

**Comment by codingwithsurya** [comment]
oops i accidentally committed a change from another issue.. please ignore

**Pull request merged**